apps/theme/styles.scss

```diff
@@ -1,3 +1,3 @@
 @use "./pageBuilder/styles/theme.scss";
 @use "./formBuilder/styles/theme.scss" as themeFormBuilder;
-@use "~@webiny/react-rich-text-renderer/styles.scss";
+@import "@webiny/react-rich-text-renderer/styles.scss";
```

Comment:
What's the difference between `@use` and `@import`, and why is `import` only used  on the last scss file, but not the first two as well?